credentials/credentials.go

```diff
@@ -334,3 +335,27 @@ func cloneTLSConfig(cfg *tls.Config) *tls.Config {
 
 	return cfg.Clone()
 }
+
+// RequestInfo contains request data to be attached to the context passed along to GetRequestMetadata calls. This API is experimental.
```

Comment:
Please put "This API is experimental" on a new line, with a blank comment line between it and the rest of the comments.  (And below; see `Bundle` in this file for an example.)

Reply:
Done.